docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md

```diff
@@ -1,15 +1,15 @@
 description: Hyperledger Besu consensus protocols
 <!--- END of page meta data -->
```

Comment:
> ./docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md:1: MD041 First line in file should be a top level header

To prevent this you have to convert file header html style comment to dashes syntax:

```
---
title: the page title
description: the page description
---

# Your first level title
```

Reply:
Ah, OK. I'll need to retrofit this back into my other PRs too.

Reply:
Metadata now uses dashes.